src/components/App.tsx

```diff
@@ -18,17 +18,17 @@ import ThemeProvider from './ThemeProvider';
 import { EvaluatedPathway } from 'pathways-model';
 import useGetPathwaysService from './PathwaysService/PathwaysService';
 import FHIR from 'fhirclient';
-import demoRecords from 'fixtures/MaureenMcodeDemoPatientRecords.json';
 import { MockedFHIRClient } from 'utils/MockedFHIRClient';
 import { getHumanName } from 'utils/fhirUtils';
 import { DomainResource, Practitioner } from 'fhir-objects';
 import styles from './App.module.scss';
-
 interface AppProps {
   demo: boolean;
+  demoId: string;
```

Comment:
We could theoretically condense these into one field... `demoId` being an optional string, if it's null then use the FHIR client. Not super high priority

Reply:
removed demo